ui/src/organizations/components/Dashboards.tsx

```diff
@@ -117,6 +118,11 @@ class Dashboards extends PureComponent<Props, State> {
     router.push(`/organizations/${params.orgID}/dashboards/import`)
   }
 
+  private summonImportFromTemplateOverlay = (): void => {
+    const {router, params} = this.props
+    router.push(`/organizations/${params.orgID}/dashboards/templates`)
```

Comment:
Same comment about the route.

Reply:
would you recommend a boolean prop instead?